addons/mail/static/tests/thread/thread.test.js

```diff
@@ -346,7 +340,6 @@ test("mark channel as fetched and seen when a new message is loaded if composer
     const env = await start();
     rpc = rpcWithEnv(env);
     await openDiscuss(channelId);
-    await focus(".o-mail-Composer-input");
```

Comment:
Hm... ok because `openDiscuss()` automatically focused the composer in the thread.
I would argue that this is a soft spec, thus I wouldn't fully rely on it. IMO the `focus()` here gives a better confidence that the conditions of the test are valid.